colors/colors.js

```diff
@@ -0,0 +1,65 @@
+/* Firefox Colors JS v1.0.3 */
+
+exports.MAGENTA_50 = '#ff1ad9';
```

Comment:
Would there be any benefit to `const`ing these, and exporting a (const) object at the bottom?

Reply:
You could `Object.freeze(exports)` at the bottom and make it immutable, but I don't think most modules make this guarantee. ES6 module exports are immutable by default, while CommonJS are not. It would make the code generation more complicated.